src/engine/Engine.ts

```diff
@@ -232,8 +232,8 @@ export class Engine extends Class {
     * Default [[IEngineOptions]]
     */
    private static _DefaultEngineOptions: IEngineOptions = {
-      width: 0,
-      height: 0,
+      canvasWidth: 0,
```

Comment:
@kamranayub's comment didn't thread, but I like his suggestion of `getDrawWidth()` over `getPixelWidth()`.